flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/expressionDsl.scala

```diff
@@ -809,6 +809,23 @@ trait ImplicitExpressionOperations {
     */
   def sha2(hashLength: Expression) = Sha2(expr, hashLength)
 
+  /**
+    * Returns the Between with lower bound and upper bound.
```

Comment:
Add a new line here and describe whether the bound values are included in the interval.